content/en/docs/reference/instrumentation/zpages.md

```diff
@@ -0,0 +1,61 @@
+---
+reviewers:
+- richabanker
+title: Kubernetes z-pages
+content_type: reference
+weight: 20
```

Comment:
Is this right? I'd put it last in the section.

Reply:
Moved to the last section. @dashpole added you as the reviewer, could you take a look please?

Reply:
```suggestion
weight: 60
```
This will make it appear last in the list.